fuzzers/005-tilegrid/generate_full.py

```diff
@@ -224,6 +224,13 @@ def process_bram_dsp():
                     segtype=tile_type.lower().replace("_", "%d_" % k, 1),
                     baseaddr=baseaddr)
 
+        def process_int():
+            if "INT" not in tile_type:
```

Comment:
simplify this if else assert to a single assert line

Reply:
Done